test/support/pages/SettingsPage.js

```diff
@@ -7,7 +7,7 @@ define(function (require) {
   var Promise = require('bluebird');
   var Common = require('./Common');
 
-  var defaultTimeout = 60000;
+  var defaultTimeout = 90000;
```

Comment:
@LeeDr because Travis and Jenkins are really slow, and we've already bumped up against the 60 second timeout for the server being stuck on the status page for more than 60 seconds.

I'm fine with adding more timeouts, and I agree that some should probably be longer than others. However, I don't think it's something we should do in this PR, and there's really no negative side effect of having all the timeouts be very long - the tests don't take longer as a result, they just fail less, which is a positive thing.